extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest: a global variable set true in Optimizely for 50% of users who visit ja wikia.
+ * window.wgGoogleSearchParam: Search ID set in Optimizely.
+ */
+$(function () {
+	'use strict';
+	if ( window.wgGoogleSearchTest ) {
```

Comment:
It would save some indenting if this checked for `!window.wgGoogleSearchTest` and then returned.